From 70edad0ad846cee725901ef7048bbb798a998746 Mon Sep 17 00:00:00 2001
From: dkl <dkl@rock-chips.com>
Date: Wed, 14 May 2014 14:05:16 +0800
Subject: [PATCH] ddrfreq && dvfs: change the notify events listened to.

Change the notify events listened to from RK_CLK_PD_PRE_ENABLE/DISABLE
to RK_CLK_PD_PREPARE/UNPREPARE.
---
 arch/arm/mach-rockchip/ddr_freq.c  | 4 ++--
 drivers/cpufreq/rockchip-cpufreq.c | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-rockchip/ddr_freq.c b/arch/arm/mach-rockchip/ddr_freq.c
index 9c45f6a2b93f..9daf0b5833ef 100644
--- a/arch/arm/mach-rockchip/ddr_freq.c
+++ b/arch/arm/mach-rockchip/ddr_freq.c
@@ -576,10 +576,10 @@ static struct input_handler ddr_freq_input_handler = {
 static int ddrfreq_clk_event(int status, unsigned long event)
 {
 	switch (event) {
-	case RK_CLK_PD_PRE_ENABLE:
+	case RK_CLK_PD_PREPARE:
 		ddrfreq_set_sys_status(status);
 		break;
-	case RK_CLK_PD_POST_DISABLE:
+	case RK_CLK_PD_UNPREPARE:
 		ddrfreq_clear_sys_status(status);
 		break;
 	}
diff --git a/drivers/cpufreq/rockchip-cpufreq.c b/drivers/cpufreq/rockchip-cpufreq.c
index 1f40586c8c63..1c686adaf2e4 100644
--- a/drivers/cpufreq/rockchip-cpufreq.c
+++ b/drivers/cpufreq/rockchip-cpufreq.c
@@ -413,11 +413,11 @@ static struct notifier_block cpufreq_reboot_notifier = {
 static int clk_pd_vio_notifier_call(struct notifier_block *nb, unsigned long event, void *ptr)
 {
 	switch (event) {
-	case RK_CLK_PD_PRE_ENABLE:
+	case RK_CLK_PD_PREPARE:
 		if (aclk_vio1_dvfs_node)
 			clk_enable_dvfs(aclk_vio1_dvfs_node);
 		break;
-	case RK_CLK_PD_POST_DISABLE:
+	case RK_CLK_PD_UNPREPARE:
 		if (aclk_vio1_dvfs_node)
 			clk_disable_dvfs(aclk_vio1_dvfs_node);
 		break;
-- 
2.35.3

